examples/tutorials/core_api_mnist/README.md

```diff
@@ -0,0 +1,43 @@
+# Core API MNIST Tutorial
```

Comment:
Shouldn't this be called "Core API Pytorch MNIST Tutorial"?

Since, it isn't useful for somebody who is, say, working on a Keras model.

If so, it would also need a directory name change.